android/app/src/main/java/app/organicmaps/sdk/util/Config.java

```diff
@@ -232,45 +229,77 @@ public static void setLocationRequested()
     setBool(KEY_MISC_LOCATION_REQUESTED);
   }
 
-  @NonNull
-  public static String getCurrentUiTheme(@NonNull Context context)
+  public static class UiTheme
   {
-    final String defaultTheme = context.getString(R.string.theme_default);
-    final String res = getString(KEY_MISC_UI_THEME, defaultTheme);
+    private static final String KEY_UI_THEME = "UiTheme";
+    private static final String KEY_UI_THEME_SETTINGS = "UiThemeSettings";
 
-    if (ThemeUtils.isValidTheme(context, res))
-      return res;
+    public static final String AUTO = "auto";
+    public static final String NIGHT = "night";
+    public static final String NAV_AUTO = "nav_auto";
+    public static final String DEFAULT = "default";
 
-    return defaultTheme;
-  }
+    public static boolean isValid(@NonNull String theme)
```

Comment:
It was before, but the naming is strange.
At least make it private, seems like it is used only inside Config.

Reply:
done